src/main.ts

```diff
@@ -0,0 +1,41 @@
+import * as core from '@actions/core';
+import * as installer from './installer';
+import * as auth from './authutil';
+import * as path from 'path';
+
+export async function run() {
+  try {
+    //
+    // Version is optional.  If supplied, install / use from the tool cache
+    // If not supplied then task is still used to setup proxy, auth, etc...
+    //
+    let version = core.getInput('node-version');
+    if (!version) {
+      version = core.getInput('version');
+    }
+
+    console.log(`version: ${version}`);
+    if (version) {
+      let token = core.getInput('token');
```

Comment:
Consider validating token is supplied. Marketplace template will be:

```yml
uses: actions/node@v2-beta
with:
  token: '' # The auth token
```

In checkout, I have seen customers with `token: ''` i think it's related to the marketplace copy/paste